Python/aot_ceval.c

```diff
@@ -105,7 +105,11 @@ static PyObject * unicode_concatenate(PyThreadState *, PyObject *, PyObject *,
 /*static*/ PyObject * special_lookup(PyThreadState *, PyObject *, _Py_Identifier *);
```

Comment:
I'm assuming that all the new code is this file is just a copy from cpython? The additions all seem reasonable to me, but if there were any changes made to the code then maybe we can call it out in some sort of way?

Reply:
Yes I tried to call them out if I really made changed but skipped some small things like removing "static"